NewsBank.js

```diff
@@ -0,0 +1,100 @@
+{
+	"translatorID": "7fc76bfc-3a1a-47e7-93cc-4deed69bee5f",
+	"label": "NewsBank",
+	"creator": "Reuben Peterkin",
+	"target": "https?://infoweb.newsbank.com/apps/news/",
+	"minVersion": "3.0",
+	"maxVersion": "",
+	"priority": 100,
+	"inRepository": true,
+	"translatorType": 4,
+	"browserSupport": "gcsibv",
+	"lastUpdated": "2021-08-06 07:05:28"
+}
+
+function detectWeb(doc, url) {
+	if (getRISText(doc)) return "newspaperArticle";
+	if (url.includes("results")) return "multiple";
+	//	Zotero.debug(url.indexOf("results"));
+	return false;
+}
+
+function getSearchResults(doc) {
+	var items = {}, found = false;
+	var rows = doc.getElementById('search-hits-gnus-search-hits-pane');
+	if (!rows) return false;
+	rows = rows.getElementsByTagName('article');
```

Comment:
For clarity:

```suggestion
	var rows = doc.querySelectorAll('#search-hits-gnus-search-hits-pane article');
```

That said, the ID feels a bit fragile. Will there be other `article` tags on the page?